UnitySetup/UnitySetup.psm1

```diff
@@ -1870,7 +1870,22 @@ function Start-UnityEditor {
             $process.Start() | Out-Null
 
             if ( $Wait ) {
-                $process.WaitForExit()
+                if( $LogFile ) {
+                    New-Item -Path $LogFile -ItemType File -Force
+                    $ljob = Start-Job -ScriptBlock { param($log) Get-Content "$log" -Wait } -ArgumentList $LogFile
+
+                    while ( -not $process.HasExited -and $ljob.HasMoreData )
+                    {
+                        Receive-Job $ljob
+                    }
+
+                    Receive-Job $ljob
```

Comment:
Might simplify to `Receive-Job -Wait -AutoRemove`

Reply:
jobs are very new to me. I'll try it out and see what I get

Reply:
your suggestion causes the task to hang indefinitely.